infores_catalog.yaml

```diff
@@ -2454,6 +2454,20 @@ information_resources:
     consumed_by:
       - infores:biothings-hpo
       - infores:mydisease-info
+  - id: infores:icd10cm
+    status: released
+    name: ICD10 Clinical Modification
+    xref:
+      - https://icd10cmtool.cdc.gov/
+    knowledge_level: knowledge_assertion
+    agent_type: not_provided
+  - id: infores:icd11-foundation
+    status: released
+    name: ICD11 Foundation
```

Comment:
you can add the obo-db-ingest products

https://github.com/biopragmatics/obo-db-ingest/tree/main/export/icd11